_data/server_permissions.yaml

```diff
@@ -0,0 +1,151 @@
+# general permissions
```

Comment:
while I realize that these files are source to generate human-readable documentation, I wonder if we want to make the source a little more readable for the sake of the maintainers? In particular, jamming all the permissions/descriptions together is harder to read. I'd favor some whitespace between items

Reply:
I agree. Done in 339a91b.